packages/quick-edit/README.md

```diff
@@ -0,0 +1,36 @@
+# VSCode for Web
+
+This package contains Cloudflare's fork VSCode for Web, to support web editing of Workers. Before working with this package, you'll need to run `./setup.sh` in `packages/quick-edit`, which will clone VSCode (currently v1.7.6), apply the patches specified in `./patches`, and symlink the top level packages within `workers-sdk`. This package primarily contains code and setup for _building_ VSCode for web, but it can also be used for development.
+
+## Developing
+
+1. You must switch your node version to Node 16 (using a tool like nvm). VSCode's build process requires this.
+2. Install dependencies (run all these commands in `packages/quick-edit`):
+
+   **`cloudflare-logic`**: `npm --prefix web/cloudflare-logic install`
+
+   **`vscode`**: `yarn --cwd vscode`
```

Comment:
perhaps it would be good to add `"preinstall"`/`"install"` to `package.json#scripts` to automate running `./setup.sh` and installing the deps for `cloudflare-logic` and `vscode`?